assembly-plugin-boilerplate/pom.xml

```diff
@@ -34,7 +34,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-project-info-reports-plugin</artifactId>
-                    <version>3.0.0</version>
+                    <version>3.1.1</version>
```

Comment:
should this use `maven-project-info-reports-plugin.version`?

Reply:
this pom file doesn't have access to that property because it doesn't inherit from the common pom.